server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyToResponseBody.scala

```diff
@@ -76,6 +106,22 @@ class NettyToResponseBody extends ToResponseBody[NettyResponse, NoStreams] {
       pipe: streams.Pipe[REQ, RESP],
       o: WebSocketBodyOutput[streams.Pipe[REQ, RESP], REQ, RESP, _, NoStreams]
   ): NettyResponse = throw new UnsupportedOperationException
+
+  private def toPart(data: Any, contentType: Option[String], name: String, filename: Option[String]): ByteBuf = {
+    val boundary = UUID.randomUUID.toString
+    val fileNameStr = filename.map(name => s"""filename="$name";""").getOrElse("")
+    val contentTypeStr = contentType.map(ct => s"Content-Type: $ct").getOrElse("")
+    val textPart =
+      s"""
+      $boundary
+          $contentTypeStr
+          Content-Disposition: form-data; $fileNameStr name="$name"
```

Comment:
I think here we should be able to add more headers ... however, maybe the names/file names would need some escaping?